docs/installation/tps/Installing_TPS_Clone.md

```diff
@@ -6,76 +6,65 @@ Overview
 
 This page describes the process to install a TPS subsystem as a clone of an existing TPS subsystem.
 
```

Comment:
Although it's obvious, it might be a good idea to mention that it's only for cloning a TPS system that have its system certs on a soft token.

Reply:
It actually makes sense since the procedure requires exporting the certs & keys into a p12 file. I updated the text.